moodler/moodle.py

```diff
@@ -155,6 +161,9 @@ def export_materials(course_id, folder):
                         f.write(assign.description)
                 for attachment in assign.attachments:
                     download_file(attachment, section_folder)
+            else:
+                logger.warn("Skipped export from unknown module '%s'",
```

Comment:
![Screenshot from 2020-08-06 16-47-39](https://user-images.githubusercontent.com/60084360/89511575-b10b5480-d804-11ea-9306-14cdcff2d14b.png)

This is from the official documentation of the `logging` module